pysrc/SBBox.cpp

```diff
@@ -35,36 +35,40 @@ namespace galsim {
     struct PySBBox 
```

Comment:
Why does module.cpp in this directory not have to be updated to include the TopHat?